modules/engage-paella-player/src/main/paella-opencast/plugins/es.upv.paella.opencast.loader/localization/da-DK.json

```diff
@@ -1,8 +1,4 @@
 {
 	"Error loading video! No video tracks found": "Videoindlæsningsfejl! Igen videospor fundet",
-	"Error loading video {0}": "Videoindlæsningsfejl {0}",
```

Comment:
Are these translations not managed by Crowdin? Otherwise you would need to change only en_US.

Reply:
Before you asked that I was sure that these translations were not managed by crowdin, but you made me check and they indeed are :o